exampleFolder/index.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+var app = express();
+const axios = require('axios');
+const { response } = require('express');
+
+//POST Route (/addPerson)
+app.post('/addPerson', (req, res) => {
+    console.log(req.body)
+    const body = req.body
+
+    axios.post('http://java-sample-api-2020.herokuapp.com/addPerson',body)
+    .then((res) => {
+        console.log(res.data)
+        res.status(200).json(res.data)
+    });
+
+    //Asynchronous Error Handling
+    Promise.resolve().then(() => {
+        throw new Error('Broken')
+    }).catch(next)
+});
+
+//DELETE Route (/deletePerson/:id)
+app.delete('/deletePerson/:id', (req, res) => {
+    const { id } = req.params;
+
+    axios.delete('https://java-sample-api-2020.herokuapp.com/deletePerson?id=1', id)
+    .then((res) => {
+        console.log(res.data)
+        res.status(200).json(res.data)
+    });
+
+    //Asynchronous Error Handling
+    Promise.resolve().then(() => {
+        throw new Error('Broken')
+    }).catch(next)
+});
+
+//GET Route (/getAllPeople)
+app.get('/getAllPeople', (req, res) => {
+    axios.get('http://java-sample-api-2020.herokuapp.com/getAllPeople')
+    .then((res) => {
+        console.log(response)
```

Comment:
'response' should be defined in the .then parameters. Initializing 'res' after previously initializing it in the .get function parameters, will create a error when calling 'res.json()' function calls in the body of the .then function.

Reply:
Fixed

Reply:
Thanks. Only problem is 'Response' in the parameters is capitalized and the 'response' in the body is not. So theres is no relation between the 2 because of the capitalization. It is best practice to use lowercase in the this situation as well.

Reply:
I think that its not actually sending anything back to the user here as well. Have you tried running your API and making requests to see what you get?